templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+Feature levels 157-159 are reserved for future use in 6.x maintenance
+releases.
+
+## Changes in Zulip 5.0
+
+**Feature level 156**
+
+No changes; feature level used for Zulip 6.0 release.
```

Comment:
I’m even less sure why we do this.

Reply:
I'm not entirely sure either, but I think we can stop doing it with 7.0 rather than 6.0 if we decide to stop; probably worth an "api design" thread, and then we can either remove it from our release checklist or document the reasoning for why we do it.